publication-commons/src/main/resources/features/publish_publication.feature

```diff
@@ -0,0 +1,74 @@
+Feature: Publish a Publication
+
+  Scenario: A User without required permissions tries to publish a Publication
+    Given a User wants to publish a Publication
+    But they are missing the required permissions
+    When they set the Accept header to "application/json"
+    And they set the Authentication header to a Bearer token with their credentials
+    And they request PUT /publication/{identifier}/publish
+    Then they receive a response with status code 401
+    And they see that the response Content-Type header is "application/problem+json"
+    And they see that the response body is a problem.json object
+    And they see the response body has a field "title" with the value "Unauthorized"
+    And they see the response body has a field "status" with the value "401"
+
+  Scenario Outline: The <User> publishes a Publication
+    Given the <User> wants to publish a Publication
+    When they set the Accept header to "application/json"
+    And they set the Authentication header to a Bearer token with their credentials
+    And they request PUT /publication/{identifier}/publish
+    Then they receive a response with status code 202
+    And they see that the response Content-type is "application/json"
+    And they see that the response Location header is the URI to the Publication
+    And they see the response body has a field "message" with the value "Publication is being published. This may take a while."
+
+    Examples:
+      | User |
+      | Owner |
+      | Curator |
+      | Editor |
+
+  Scenario: The Owner publishes a Publication with a malformed identifier
+    Given the Owner wants to publish a Publication
+    When they set the Accept header to "application/json"
+    And they set the Authentication header to a Bearer token with their credentials
+    And they request PUT /publication/{malformed identifier}/publish
+    Then they receive a response with status code 400
+    And they see that the response Content-Type header is "application/problem+json"
+    And they see that the response body is a problem.json object
+    And they see the response body has a field "title" with the value "Bad Request"
+    And they see the response body has a field "status" with the value "400"
+
+  Scenario: The Owner checks status on a published Publication
```

Comment:
I think this is a misnamed — they're resubmitting the request on an already submitted request. While this has no effect, in terms of semantics, I'd look at the location and check the status if I was "checking"

Reply:
I guess this requires a feature for checking the Location URI and asserting that the status has changed